ios/Classes/InAppWebView.swift

```diff
@@ -1040,6 +1040,9 @@ public class InAppWebView: WKWebView, UIScrollViewDelegate, WKUIDelegate, WKNavi
                     if !action.description.starts(with: "onContextMenuActionItemClicked-") && contextMenuOptions.hideDefaultSystemContextMenuItems {
                         return false
                     }
+                    if !contextMenuOptions.filterDefaultSystemContextMenuItems != nil && !contextMenuOptions.filterDefaultSystemContextMenuItems.contains(action.description) {

```

Comment:
This line is hard to understand. Isn't `!xxx != nil` equals to `xxx == nil`? And if it's `nil`, the second condition seems completely useless.

```suggestion
                    if contextMenuOptions.filterDefaultSystemContextMenuItems != nil && !(contextMenuOptions.filterDefaultSystemContextMenuItems?.contains(action.description) ?? false) {

```

Reply:
I am pretty new to Swift and cannot really understand your correction, but i suppose it is valid, so I let Lorenzo decide, I hope that is ok.